### ISSUE TITLE
Screen reader is not announcing any information about the graph in scan mode under summary 

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/azuredatastudio#22753]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio;#Win32; #WCAG1.3.1; #NVDA; #ScreenReader; #Benchmark;#JAWS;#AILimited; #Regressed:05-30-23;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**
### Environment and OS details:
Application name: Assessment
App version: Azure Data Studio 1.39.1
Windows Version: Win11
Screen reader: NVDA [version: 2022.2], JAWS [version: 2022.2207.25 ILM]
### Pre-Requisite:

1. Download the azure data studio from the link https://docs.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver15
2. Once installation done , Install Database Migration Assessment for Oracle Extension in Azure data studio
3. Tab Till new connection and press enter
4. Select connection type as oracle
5. Enter a valid server name username and password and press connect
### Repro Steps

1. Launch Azure Data Studio.
2. Connect to server.
5. Tab to the Migration Assessment button and then press enter. "Database Migration Assessment for Oracle" page will be opened.
6. Tab to the "Run New Assessment" button at top the page and press enter. New Database Migration Assessment for Oracle dialog box will open. Fill the correct details. Tab to "Start Assessment button and press enter.
7. New Assessment can be seen in last five assessment tab. Tab to the assessment press enter. New tab will get open.
8. Tab to Assessment overview by platforms section. Tab to Assessment for Azure virtual machine in the section. Tab to view report details button and press enter. New screen will open.
9. Tab to "Summary" tab and press enter.
10. Start NVDA and on scan mode and press down arrow key to navigate graph
11. Observed that Screen reader screen reader is not announcing the information of graph
### Actual Result:
Screen reader is not announcing any information about the graph in scan mode under summary
Note: issue is seen with Jaws
### Expected Result:
when navigated to the graph the information we see visually should be reader out  or there can be an alternative way the image can be set
### Attachment:
![Presentation1](https://user-images.githubusercontent.com/124254981/232485187-5a5abaa9-c695-45bd-ab2c-57a9864adc39.gif)

**Comment by kapilvaishna** [comment]
Primary bug: https://msdata.visualstudio.com/Migration%20Assistant/_workitems/edit/2081174

**Comment by P-Gaharwar** [comment]
@Charles-Gagnon \ @kburtram: Can you please triage this bug?

**Comment by alanrenmsft** [comment]
@stuti149 please work with the tester to get the issue closed.

**Comment by alanrenmsft** [comment]
@stuti149 could you please take a look at this issue?

**Comment by stuti149** [comment]
@kapilvaishna , the chart here is an image component and information depicted by this chart is "Chart showing the number of Oracle schema objects that can be converted to {SQL,PGSQL} and those that cannot be converted, categorized by object type". And I see that this info is being read in the attachment.

**Comment by kapilvaishna** [comment]
@stuti149 Here all information present in chart like simple if code object is from 1 to 100 Should be available for screen reader user which is shown through chart or should be available on page in text form.
 if issue is not clear please feel free to ping me offline

**Comment by kisantia** [comment]
@stuti149 fyi I'm moving this issue from the July release to the September release

**Comment by stuti149** [comment]
@kapilvaishna I am closing this issue as it is resolved and tested by Accessibility team on DMAO extension

**Comment by kapilvaishna** [comment]
Issue is not repro in ADS Insiders v1.45.0 and azuredatastudio-dma-oracle-0.1.0 (1).vsix (available in primary bug)
So, closing the bug 

https://github.com/microsoft/azuredatastudio/assets/124254981/2fc32da6-63be-4dad-bf80-d0ad5b7e082d
